addons/point_of_sale/models/pos_session.py

```diff
@@ -542,7 +542,7 @@ def get_income_account(order_line):
         price = order_line.price_unit * (1 - (order_line.discount or 0.0) / 100.0)
         taxes = tax_ids.compute_all(price_unit=price, quantity=order_line.qty, currency=self.currency_id, is_refund=order_line.qty<0).get('taxes', [])
         date_order = order_line.order_id.date_order
-        taxes = [{'date_order': date_order, **tax} for tax in taxes]
+        taxes = [{'date_order': date_order, **tax} for tax in taxes if abs(tax['amount']) > 0]
```

Comment:
you can delete this commit, a fix had been done for the tax at 0 % in https://github.com/odoo/odoo/commit/c7936d2884c10865bfc39a43d30aef6926f29079#diff-f0aa3e0f5e29080288e68f327654dc6c